shared-bindings/is31fl3741/__init__.c

```diff
@@ -24,16 +24,89 @@
  * THE SOFTWARE.
  */
 
+#include "shared-bindings/is31fl3741/__init__.h"
+
 #include <stdint.h>
+#include <stdbool.h>
 
 #include "py/obj.h"
 #include "py/runtime.h"
 
+#include "shared-bindings/busio/I2C.h"
 #include "shared-bindings/is31fl3741/IS31FL3741.h"
 
+
+//| """Low-level is31fl3741 implementation
+//|
+//| The `is31fl3741_write` module contains a helper method to write out bytes in
+//| over the I2C bus."""
```

Comment:
I don't think you need this because the module is `is31fl3741` and has more than the helper function.